plugin/mlsdbprovider.cpp

```diff
@@ -606,6 +612,14 @@ void MlsdbProvider::startPositioningIfNeeded()
     calculatePositionAndEmitLocation();
     quint32 updateInterval = minimumRequestedUpdateInterval();
     m_recalculatePositionTimer.start(updateInterval, this);
+
+    m_detectGpsLockConnected =
+            QDBusConnection::sessionBus().connect(
```

Comment:
If the hybris plugin is started after mls plugin this fails to connect, right? Also it feels wrong to connect this to hybris plugin after thinking it a bit.

Reply:
I agree.  Also, as noted in the commit message of that commit, from my testing it seems that geoclue master automatically disconnects clients from less-accurate positioning methods, once gps lock is acquired.  Thus, the mlsdb plugin will automatically stop itself in that case anyway.

So, I will remove this commit from the PR, as it's not needed.